src/test/resources/Feature files/login.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Login feature
+
+  Background:user opens URL 'https://www.saucedemo.com/'
+  @ValidLogin
+  Scenario Outline: Valid username and password
+    When user input <Username> and <Password>
+    Examples:
+      | Username                | Password     |
+      | standard_user           | secret_sauce |
+      | problem_user            | secret_sauce |
+      | performance_glitch_user | secret_sauce |
+    And user click on Login button
+    Then user is successfully logged in
```

Comment:
O sa te rog ca pe viitor sa scrii cu litera mare enunturile  e.g."User is...."

Reply:
Done! Merci, asa o sa fac.